spec/index.html

```diff
@@ -1293,6 +1295,180 @@ <h3>Grammar</h3>
 
     <div data-include="turtle-bnf.html"></div>
   </section>
+
+  <section>
+    <h2>Selected Terminal Literal Strings</h2>
+    <p>This document uses some specific terminal literal strings [[EBNF-NOTATION]].
+      To clarify the Unicode code points used for these terminal literal strings, the following
+      table describes specific characters and sequences used throughout this document.</p>
+
+    <!-- Selected unicode character descriptions, taken from https://en.wikipedia.org/wiki/List_of_Unicode_characters -->
+    <table class="cp-definitions">
+      <thead>
```

Comment:
I suggest splitting the current table into 2 or maybe 3 sections (depends how many fit into the current body width) that are presented in 2 or 3 columns beneath the intro paragraph.

Reply:
That would need to be a responsive layout do deal with screen-size issues. I think it would be great, but maybe should be addressed along with other responsive UI considerations.